tests/benchmarks/conftest.py

```diff
@@ -160,3 +160,27 @@ def dataset(make_dataset):
 @pytest.fixture
 def remote_dataset(test_config):
     pytest.skip("fixme")
+
+
+@pytest.fixture
+def make_project(tmp_path_factory):
+    def _make_project(url, rev=None):
+        path = os.fspath(tmp_path_factory.mktemp("dvc-project"))
+        clone(url, path, branch=rev)
+        return path
+
+    return _make_project
+
+
+@pytest.fixture
+def project(test_config, monkeypatch, make_project):
```

Comment:
We currently manually create datasets in tests like `test_status`, but that could potentially also be turned into a project template.